### ISSUE TITLE
Add null safety support

### ISSUE THREAD

**Issue by zzterrozz** [btastic/flutter_native_image#78]

**Comment by Hwan-seok** [comment]
👍🏻👍🏻
If you do not mind, could you tell me when is the milestone?

**Comment by zzterrozz** [comment]
@dw2kim  did the job already and pushed a PR that has not been accepted yet. 

You can use tu with that way in your pubspec.yaml: 


flutter_native_image:
    git:
      url: git://github.com/dw2kim/flutter_native_image.git

**Comment by Hwan-seok** [comment]
should write branch name also.
Use
```
  flutter_native_image:
    git:
      url: git://github.com/dw2kim/flutter_native_image
      ref: feature/null-safety-support
```
instead
```
  flutter_native_image:
      git:
        url: git://github.com/dw2kim/flutter_native_image.git
```

**Comment by Hwan-seok** [comment]
@dw2kim
Thanks for applying null safety.

But one Exception occurs on [here](https://github.com/dw2kim/flutter_native_image/blob/009faa553d49b6ff7bd484383ff1c520ee95ef12/lib/flutter_native_image.dart#L62)
as 
```
Exception has occurred.
_CastError (type 'Future<dynamic>' is not a subtype of type 'FutureOr<Map<dynamic, dynamic>>' in type cast)
```

please check if available.

**Comment by btastic** [comment]
It's done now thanks to @dw2kim

**Comment by zzterrozz** [comment]
@btastic 

After `flutter clean` `flutter pub upgrade` with `flutter_native_image: ^0.0.5+3` I have the error :

```
Error: Cannot run with sound null safety, because the following dependencies
    don't support null safety:

     - package:flutter_native_image
```

**Comment by btastic** [comment]
Sorry, but that doesn't tell my anything. Maybe @dw2kim or @Hwan-seok can chime in to help?

**Comment by Hwan-seok** [comment]
@zzterrozz 
The currently published version of this library is not support null safety.
If you want null safety migrated version, take a look at [this](https://github.com/btastic/flutter_native_image/issues/78#issuecomment-815411706)

**Comment by zzterrozz** [comment]
> @zzterrozz 
> 
> The currently published version of this library is not support null safety.
> 
> If you want null safety migrated version, take a look at [this](https://github.com/btastic/flutter_native_image/issues/78#issuecomment-815411706)

You just referred my own message..

My point is that the null-safety PR that has been pushed to pub.dev seems to still have an issue.

**Comment by Hwan-seok** [comment]
Oh, I misunderstood that you are not the author.

I think new version did not pushed to pub.dev yet
Did I misunderstood this too?

**Comment by btastic** [comment]
New package is published as version: 0.0.6.

https://pub.dev/packages/flutter_native_image
